static/usage/select/objects-as-values/multiple-selection/angular/app.component_ts.md

```diff
@@ -0,0 +1,46 @@
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: 'app.component.html',
+})
+export class AppComponent {
+  currentFood = "[]";
+
+  foods = [
+    {
+      id: 1,
+      name: "Apples",
+      type: "fruit"
+    },
+    {
+      id: 2,
+      name: "Carrots",
+      type: "vegetable"
+    },
+    {
+      id: 3,
+      name: "Cupcakes",
+      type: "dessert"
+    }
+  ];
+
+  compareWith(o1, o2) {
+    if(!o1 || !o2) {
+      return o1 === o2;
+    }
+
+    if(Array.isArray(o2)) {
+      return o2.some((o) => o.id === o1.id);
+    }
+
+    return o1.id === o2.id;
+  }
+
+  handleChange(ev) {
+    this.currentFood = JSON.stringify(ev.target.value);
```

Comment:
~ Angular has a built-in `| json` pipe that can handle this for you. We should consider using that, just to remove any confusion of why the initial value is a string of an array.

Updated Stackblitz example that shows usage: https://stackblitz.com/edit/angular-v3mgxj?file=src%2Fapp%2Fapp.component.html,src%2Fapp%2Fapp.component.ts

This would apply for both Angular examples.